ecosystem/ffmpeg_plugin/README.md

```diff
@@ -178,7 +178,7 @@ ffmpeg -stream_loop -1 -i test.wav -p_port 0000:af:01.1 -p_sip 192.168.96.3 -p_t
 For pcm16 audio, use `mtl_st30p_pcm16` muxer, set `pcm_fmt` to `pcm16` for demuxer.
 
 ```bash
-ffmpeg -stream_loop -1 -i test.wav -p_port 0000:af:01.1 -p_sip 192.168.96.3 -p_tx_ip 239.168.85.20 -udp_port 30000 -payload_type 97 -ptime 1ms -f mtl_st30p_pcm16 -
```

Comment:
NO WAIT THIS IS THE WORKING ONE we have pcm16 muxer version in the receive there is a problem